cedar-policy-core/src/entities/json/context.rs

```diff
@@ -82,12 +82,12 @@ impl<'e, 's, S: ContextSchema> ContextJsonParser<'e, 's, S> {
         &self,
         json: serde_json::Value,
     ) -> Result<Context, ContextJsonDeserializationError> {
-        let vparser = ValueParser::new(self.extensions);
+        let vparser = ValueParser::new(self.extensions.clone());
```

Comment:
is it possible for `ValueParser` to hold an `&Extensions` instead?  (I do recognize that `Extensions` is still pretty cheap to clone, so this is a minor nit)